test_scripts/AppServices/UnpublishAppService/002_HMI_success_flow.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Missing UnpublishAppService tests in test set.

Reply:
Also, should probably include a test case where an app tries to unpublish a service that it didn't publish.